pyvista/plotting/renderer.py

```diff
@@ -695,6 +695,32 @@ def has_charts(self):  # numpydoc ignore=RT01
         """Return whether this renderer has charts."""
         return self._charts is not None and len(self._charts) > 0
 
+    @property
+    def mpl_charts(self):  # numpydoc ignore=RT01
```

Comment:
The name is best changed indeed.
I don't remember exactly why `_charts` was made a private attribute of `Renderer`, but it might be a good idea to add a `get_charts` method that can filter on specific chart types (e.g. `ChartMPL` for matplotlib charts only).

Reply:
In my case I don't really need to filter specific chart types but to get the list of charts.
Would it be fine to replace `mpl_charts` by `charts`?

Reply:
Alright, I would still make it a method (`get_charts`) instead of a property, such that filters could be added in the future then.

Reply:
Done, thanks!